Cython/Compiler/FusedNode.py

```diff
@@ -9,6 +9,26 @@
 from ..Utils import OrderedSet
 
 
+def _gen_unique_values():
```

Comment:
1) This is just `itertools.count()`, but:
2) We keep unique name generators like these in the `GlobalState` (or `FuncState` for function-local names), so that you get predictable names for each module, regardless of how many Cython handles at a time. See, for example, `new_const_cname()` in `Code.py`.